data/json/uncraft/armor/torso.json

```
@@ -199,5 +199,13 @@
     "time": "4 m",
     "qualities": [ { "id": "FABRIC_CUT", "level": 1 }, { "id": "CUT", "level": 2 } ],
     "components": [ [ [ "felt_patch", 38 ] ], [ [ "sheet_felt", 20 ] ], [ [ "button_plastic", 5 ] ] ]
+  },
+  {
+    "result": "cheerleader_top_short",
+    "type": "uncraft",
+    "activity_level": "LIGHT_EXERCISE",
+    "time": "2 m,30 s",
```

```suggestion
    "time": "2 m 30 s",
```

Have you checked this syntax works ingame? I never saw it used anywhere with time, so I'm a little sceptical